.github/actions/setup-binary/action.yml

```
@@ -24,6 +24,9 @@ inputs:
   install_path:
     description: 'The path on disk in which to extract the binary, this cannot be equal to the current directory. Set `add_to_path` to true to add this to the $PATH.'
     required: true
+  binary_subpath:
+    description: 'Name of binary. Needed to set as executable.'
```

this should probably be more descriptive

```suggestion
    description: 'The subpath to the binary relative to the install path. This should include the binary name (e.g. dist/binary-name). Required to set the binary as executable.'
```